static/styles/settings.css

```diff
@@ -95,6 +95,10 @@ h3 .fa-question-circle-o {
     .input-size {
         width: 214px;
     }
+    .btn-danger:hover {
+        color: hsl(0, 0%, 100%);
+        background-color: hsl(354, 70%, 54%);
+    }
 }
```

Comment:
I am not sure why we want these changes.

Reply:
Actually the hover was not working properly, hence I had to overwrite some of the css for hover under new-style class.

Reply:
Have you added these for owner case when the button is actually enabled?

Reply:
I am not sure whether we want to change the background color of the button to red when the button is enabled. Previously also, we were not changing the background to red, I think this was also a bug.

Even if we want to do this change, you should do this change in a separate commit.

Reply:
The color is only when there is a hover over the button. The change is only for the owner side and not for the admin.

Reply:
I don't think it will cause a ui change for other elements hence I made it in the same commit.

Reply:
However I will make a new commit for this change to avoid any ui bug. Thanks for the review.

Reply:
Can you post a before/after screenshot for this piece?